pkg/core/region.go

```diff
@@ -360,12 +360,12 @@ func (r *RegionInfo) GetPeer(peerID uint64) *metapb.Peer {
 	return nil
 }
 
-// Role is the role of the region.
-type Role int
+// Rule is the rule for balance range scheduler
```

Comment:
why change name by mistake?

Reply:
the design used the rule not role. see https://github.com/pingcap/tidb/issues/59681

Reply:
Are there any other similar examples where "rule" is used? It would be best to standardize their terminology.

Reply:
The keyword 'rule' is a new syntax.https://github.com/pingcap/tidb/pull/60027/files#diff-842be04cb2c39aecf993b3654455d163e16e9603185ce63871f6842a04c04574